src/pages/en/guides/styling.md

```diff
@@ -176,6 +176,166 @@ You can also use the `<link>` element to load a stylesheet on the page. This sho
 
 Because this approach uses the `public/` directory, it skips the normal CSS processing, bundling and optimizations that are provided by Astro. If you need these transformations, use the [Import a Stylesheet](#import-a-local-stylesheet) method above.
 
+## Cascading Order
+
+When evaluating multiple CSS rules that apply to the same element, the browser uses _specificity_ and _order of appearance_ to determine which style to show. If one rule is more _specific_ than another, it will take precedence:
+
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: red }
+  div > h1 {
+    color: purple
+  }
+</style>
+<div>
+  <h1 class="red">
+    This header will be purple!
+  </h1>
+</div>
+```
+
+If two rules have the same specificity, the _order of appearance_ is evaluated, and the last rule will take precedence:
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: purple }
+  h1 { color: red }
+</style>
+<div>
+  <h1 class="red">
+    This header will be red!
+  </h1>
+</div>
+```
+
+Among Astro features, if the specificity of a rule is the same, the resulting styles will appear in this order:
+
+- **`<link>` tags in the head** (lowest precedence)
+- **imported styles**
+- **scoped styles** (highest precedence)
+
+### Scoped Styles 
+
+Using [scoped styles](#scoped-styles) does not increase the _specificity_ of your CSS, but it will always come last in the _order of appearance_. They will therefore take precedence over other styles of the same specificity. For example, if you import a stylesheet with the same rule as a scoped style, the scoped style will apply:
+
+```css title="make-it-purple.css"
+h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be red!
+  </h1>
+</div>
+```
+
+If you make the imported style _more specific_, it will have higher precedence over the scoped style:
+
+```css title="make-it-purple.css"
+div > h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+### Import Order
+
+When importing multiple stylesheets, if any styles have the same specificity, the _last one imported_ wins:
```

Comment:
```suggestion
When importing multiple stylesheets in an Astro component, the CSS rules are evaluated in the order that they are imported. A higher specificity will always determine which styles to show, no matter when the CSS is evaluated. But, when conflicting styles have the same specificity, the _last one imported_ wins:
```
Something like this. I wanted to (a) give this a bit more (even if repeated) context in case anyone jumps to this section right away. And (b) explain what's happening in terms of the order of evaluation (since that's the key factor here). If they _have_ been reading diligently top to bottom, then they KNOW order of evaluation matters when specificity is the same. What they need to know in this section is "In which order does ASTRO evaluate multiple stylesheets?" and I prefer to say that explicitly, matching the context we've set above.

Reply:
I like this, thanks!